### DISCUSSION TITLE
A few suggestions and improvements

### DISCUSSION THREAD

**Discussion by opt8** [YimMenu/YimMenu#3513] [General]
hi, ive been using this menu for a while and have collected some list of errors, suggestions and wishes

1. "creating a squad", second option in world section - created by this function npc dont use weapons after target was killed, except jets (they drive vehicle but weapon is deleted - they show finger, lol, helicopter just flying and etc)
2. there are no ways to find out who spawn (attacking) npc - if possible would be great if these models were highlighted in esp together with the name of creator or something like that
3. in selected player "transport" section, does not work every "special transport ability" except push to the left (maybe I misunderstand how it should work, but the push to the right, compared to the push to the left, does not work surely)
4. in selected player "other" section, button "enter interior" works not every time - as I noticed, only if the player is in interiors not below ground level.
5. in selected player "other" section, semigode checkbox does not prevent other player death by explosion or falling from a height or something like that, i think it happens when damage is taken more than player had hp
6. in car handling editor - i cant create all wheels drive, changing the weight of the car doesn't work too. there are no special fields for editing planes and helis handling
7. we have restricting weapon - but no such function in selected player menu
8. we can only hijack train, but not tram in the city (maybe would be fun?), anyway train control does not work
9. not all missions in network session are recognized by menu - many are impossible to skip or start (only using time warp but in this situation many players can be kicked)

**Comment by lonelybud** [comment]
> there are no ways to find out who spawn (attacking) npc - if possible would be great if these models were highlighted in esp together with the name of creator or something like that

i want to code something related to it like keeping track of synced entities upto some limit for peds, vehicles, object & pickups. Along with explosion detection this will help improve modder detection. Then we can group them all under a setting called higher level detections bla bla.
